pkgs/development/libraries/kmsxx/default.nix

```diff
@@ -1,25 +1,36 @@
-{ stdenv, fetchFromGitHub, cmake, pkgconfig, libdrm, python }:
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, libdrm
+, libGL, mesa, libX11, wayland
+, withPython ? false, python }:
 
 stdenv.mkDerivation {
   pname = "kmsxx";
-  version = "2018-10-23";
+  version = "2019-10-02";
 
   src = fetchFromGitHub {
     owner = "tomba";
     repo = "kmsxx";
     fetchSubmodules = true;
-    rev = "c0093c91f0fa2fd6a5b9d1b206a6f44dcd55bfb5";
-    sha256 = "03rv92r938nxb4k4gwcvxy76jnhxdx6x60b58jws83285hd9rgkf";
+    rev = "934416e1dfa8222a0a1746a5a0951e2b08d02eae";
+    sha256 = "1zp6xxls9f5y2s1i51fnlgw124zxvgx3rys8yci4bi4qc59brpn1";
   };
 
   enableParallelBuilding = true;
 
+  cmakeFlags = [ "-DKMSXX_ENABLE_KMSCUBE=ON" ]
+    ++ stdenv.lib.optional (!withPython) "-DKMSXX_ENABLE_PYTHON=OFF";
+
   nativeBuildInputs = [ cmake pkgconfig ];
-  buildInputs = [ libdrm python ];
+  buildInputs = [ libdrm python
+    libGL mesa libX11 wayland ];
 
   pythonPath = [ ];
   passthru.python = python;
 
+  preConfigure = ''
+    echo "install(TARGETS kmscube DESTINATION bin)" >> kmscube/CMakeLists.txt
+    echo "install(TARGETS kmsview kmscapture kmsblank wbcap wbm2m)" >> utils/CMakeLists.txt
+  '';
```

Comment:
should this be upstreamed?

Reply:
@jonringer Sure! https://github.com/tomba/kmsxx/pull/38

Reply:
@jonringer Author said that these utilites are not polished to ship, so I removed these lines